DigitalLearningSolutions.Web/Styles/trackingSystem/sideNavMenus.scss

```diff
@@ -0,0 +1,49 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+.side-nav-menu {
+  @include govuk-media-query($until: desktop) {
+      display: none;
+  }
+}
+
+.side-nav__heading {
+  font-weight: 600;
+  color: #4c6272;

```

Comment:
We should probably be using the nhs defined colours for these rather than the explicit codes for them e.g. this one is $color_nhsuk-grey-1 (or various other specific names that can be found [here](https://service-manual.nhs.uk/design-system/styles/colour)